src/routes/local.ts

```diff
@@ -15,27 +19,33 @@ const PROPS_TO_SEND_IN_RESPOSNE = [
   "purpose",
 ];
 
-router.get("/two-line-elements/:id", (req, res) => {
-  if (req.params.id === undefined) return res.sendStatus(400);
-  n2yo
-    .getTwoLineElements(req.params.id)
-    .then((result) => res.json(result))
-    .catch(() => res.sendStatus(400));
-});
+router.get("/two-line-elements/:id",
+  (req, res, next) => satelliteIdRouteValidation(req, res, next),
```

Comment:
I think that it's worth to explain why does it happen. 
You pass a callback here. Callback is nothing more than an argument that's a function. So instead of our usual objects, strings, numbers, arrays we pass function. 
What usually happens is that the callback is being taken, then the function which takes callback (here it's the `get` function attached to the object `router`) takes the function, passes certain arguments to it and executes it. 
So I. E. 
```
function myCallbackTakingFunction(callback) {
    const a = 2
    const b = 4
    callback(a, b) 
}
```
This is an example how callbacks usually looks like. 
What you do here is to pass
`(a, b) => myFunction(a, b) ` and pass it to the callback taker so:
`myCallbackTakingFuction((a, b) => myFunction(a, b))`
But what really is the `(a, b) => myFunction(a, b) ` here? 
It's a function, that's a wrapper for myFunction.
You pass this function as a callback argument then.
You essentially create here a function that takes 
arguments a and b, and then pass them to myFunction.  
Which means that
`(a, b) => myFunction(a, b)` and `myFunction(a, b) ` gives the same effect. 

If our callback taker will pass a and b to our wrapper, the function will behave the same way, 
As if you'd just let him pass a and b directly to myFunction, so you can just give him myFunction here. 

Now it doesn't always works, and understanding why it works is crucial to understand why it might not work. 

It might not work if your function uses `this` or if the callback taker does some voodoo magic on your function. 
But in most cases if the callback taker passes exact amount of arguments and you want to pass all of them, there's no need for a wrapper.